rules-reviewed/camel3/camel2/java-multiple-camelcontexts-per-application-not-supported.windup.xml

```diff
@@ -32,5 +32,23 @@
                 <matches pattern="(BeanInject|Consume|DynamicRouter|EndpointInject|Produce|PropertyInject|RecipientList|RoutingSlip)" />
             </where>
         </rule>
+        <rule id="java-multiple-camelcontexts-per-application-not-supported-00001">
+            <when>
+                <javaclass references="org.apache.camel.cdi.{CdiCamelContextName}">
+                    <location>IMPORT</location>
+                </javaclass>
+            </when>
+            <perform>
+                <hint title="{CdiCamelContextName} has been removed" effort="2" category-id="mandatory">
```

Comment:
For efforts (i forgot to tell you, sorry) we use some fixed values for which we provide documentation [How Story Points are Estimated in Rules](https://access.redhat.com/documentation/en-us/red_hat_application_migration_toolkit/4.3/html-single/rules_development_guide/index#how_story_points_are_estimated_in_rules).
Hence i proposed to change the effort to `3`.
```suggestion
                <hint title="{CdiCamelContextName} has been removed" effort="3" category-id="mandatory">
```